src/idl_gen_general.cpp

```diff
@@ -203,8 +203,13 @@ class GeneralGenerator : public BaseGenerator {
       const std::string &classcode, bool needs_includes) {
     if (!classcode.length()) return true;
 
-    std::string code;
-    code = code + "// " + FlatBuffersGeneratedWarning();
+    std::string code("//------------------------------------------------------------------------------\n"
```

Comment:
Should we also add the [GeneratedCodeAttribute](https://msdn.microsoft.com/en-us/library/system.codedom.compiler.generatedcodeattribute(v=vs.110).aspx) to the generated classes?

But then the problem would be what version to put in. Does Flatbuffers store a version somewhere?